more/pixels/gallery.js

```diff
@@ -245,9 +245,13 @@ export function makeGallery(
     const useRightAmount = useRightAssay.make(pixelList);
     const transferRightAmount = transferRightAssay.make(pixelList);
 
-    pixelMint.destroy(pixelAmount);
-    useRightMint.destroy(useRightAmount);
-    transferRightMint.destroy(transferRightAmount);
+    try {
+      pixelMint.destroy(pixelAmount);
+      useRightMint.destroy(useRightAmount);
+      transferRightMint.destroy(transferRightAmount);
+    } catch (err) {
+      console.log(err);
```

Comment:
Before this PR, when destroy happened and the amount wasn't found, it just returned with no error. I changed that logic to throw an error, but that necessitated a change here to handle it gracefully (this happens every time the gallery tries to revoke a pixel at the start). 

All of these lines go away in https://github.com/Agoric/ERTP/pull/36 so this should be a very temporary change to handle the exceptions that won't exist in the next few days.

Reply:
So I think in answering the questions, I would have to answer in the context of both this change, and the change to `pixelMintController.destroy`:

try-catch with a log message will log that the amount didn't exist, where previously, there was no log message and no exception.

It doesn't prevent execution from stopping, and neither did the code before this PR

It makes the exceptions more visible in debugging (because they didn't exist before), and for normal running and our test framework, it is the same (because normal running isn't affected by the logs, and our test framework uses a different logging tool than console.log for the test logs)

Reply:
thanks. That seems fine, then.